tests/ctst/package.json

```diff
@@ -20,7 +20,7 @@
     "@typescript-eslint/eslint-plugin": "^5.45.0",
     "@typescript-eslint/parser": "^5.45.0",
     "babel-jest": "^29.3.1",
-    "cli-testing": "github:scality/cli-testing.git#0.3.1",
+    "cli-testing": "github:scality/cli-testing.git#dd6bb9ca6dccb3fe38cf6c156dcbc208f22c7ca8",
```

Comment:
Adding this as a reminder to change tag to next release of clitesting

Reply:
Yes, done post release of ctst 0.4.1